src/test/java/io/github/perrymant/moneymaker/FileIOTest.java

```diff
@@ -0,0 +1,36 @@
+package io.github.perrymant.moneymaker;
+
+import org.junit.Test;
+
+import java.io.IOException;
+
+import static org.junit.Assert.assertEquals;
+
+public class FileIOTest {
+
+    @Test
+    public void canReadData() throws IOException {
+        String fileName = "src/test/java/io/github/perrymant/moneymaker/file_with_data";
```

Comment:
sounds like this file should be in `src/test/resources` instead. Applies throughout